settings.gradle

```diff
@@ -0,0 +1,17 @@
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+            url "http://artifactory.tools.intole.net/artifactory/extole-packages/api-stubs"
+            allowInsecureProtocol true
+        }
+        maven {
+            url "http://artifactory.tools.intole.net/artifactory/extole-packages/"
```

Comment:
Please add a TODO with ticket number as clients can't use this

Reply:
added: `// TODO to be replaced by Github Packages ENG-15109`